### ISSUE TITLE
[BITV] 9.1.4.3/9.1 - The heading "Groups" does not meet the contrast requirements for text, as the contrast ratio with the background is only 3,3:1. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41850]
![image](https://github.com/nextcloud/server/assets/6078378/77bbaddd-4b08-4433-b768-2800a9b1f05b)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-6fe46b78da-v9-n1

**Comment by susnux** [comment]
Waiting for next nextcloud-vue update, was fixed in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4885
